Docs/src/manual.md

```diff
@@ -885,3 +886,77 @@ One example of this is the Ogre::Light class. It extends AnimableObject and prov
 ## AnimableValue
 
 When implementing custom animable properties, you have to also implement a number of methods on the AnimableValue interface - basically anything which has been marked as unimplemented. These are not pure virtual methods simply because you only have to implement the methods required for the type of value you’re animating. Again, see the examples in Light to see how this is done.
+
+
+@page Instancing Instancing
+
+Instancing significantly reduces the CPU overhead of submitting many separate draw calls and is a great technique for rendering trees, rocks, grass, RTS units and other groups of similar (but necessarily identical) objects.
+
+OGRE supports a variety of techniques to speed up the rendering of many objects in the Scene.
+
+<dl compact="compact">
+<dt>@ref Static-Geometry</dt> 
+<dd>Pre-transforms and batches up meshes for efficient use as static geometry in a scene.</dd>
+<dt>@ref Instance-Manager</dt> 
+<dd>Instancing is a way of batching up geometry into a much more efficient form, but with some limitations, and still be able to move & animate it.</dd>
+</dl>
+
+@tableofcontents
+
+# Static Geometry {#Static-Geometry}
+Modern graphics cards (GPUs) prefer to receive geometry in large batches. 
+It is orders of magnitude faster to render 10 batches of 10,000 triangles than it is to render 10,000 batches of 10 triangles, even though both result in the same number of on-screen triangles.
+
+Therefore it is important when you are rendering a lot of geometry to batch things up into as few rendering calls as possible. 
+This class allows you to build a batched object from a series of entities in order to benefit from this behaviour. Batching has implications of it's own though:
+ - Batched geometry cannot be subdivided; that means that the whole group will be displayed, or none of it will. This obivously has culling issues.
```

Comment:
```suggestion
 - A geometry region cannot be subdivided; that means that the whole group will be displayed, or none of it will. This obivously has culling issues.
```